README.md

```diff
@@ -189,6 +189,77 @@ be repeated with just `user`, `block_usage`, and `file_usage` changing.*
 
 *Warning: A block must be equal to 1 KB for proper conversions.*
 
+### Balance Warnings
+
+We currently support displaying warnings to users on the Dashboard if their
+balance is nearing its limit. This requires an auto-updated (it is
+recommended to update this file every **15 minutes** with a cronjob) JSON file
+that lists all user balances. The JSON schema for version `1` is given as:
+
+```json
+{
+  "version": 1,
+  "timestamp": 1525361263,
+  "config": {
+    "unit": "RU",
+    "project_type": "project"
+  },
+  "balances": [
+    {
+      ...
+    },
+    {
+      ...
+    }
+  ]
+}
+```
+
+Where `version` defines the version of the JSON schema used, `timestamp`
+defines when this file was generated, and `quotas` is a list of quota objects
+(see below).
+
+The value for `config.unit` defines the type of units for balances and `config.project_type` would be project, account, or group, etc.
```

Comment:
You should explicitly state that this value is used in the locale files, and so can be any string value that makes sense.

Reply:
Updated to call both out.